src/Collection/Fulfilment/OrderCollection.php

```diff
@@ -133,6 +135,25 @@ private function getShipmentOptions(AbstractDeliveryOptionsAdapter $deliveryOpti
         return $options;
     }
 
+    /**
+     * @param  \MyParcelNL\Sdk\src\Model\Consignment\DropOffPoint $dropOffPoint
+     *
+     * @return array
+     */
+    private function getDropOffPoint(DropOffPoint $dropOffPoint): array
```

Comment:
Het is (blijkbaar) de bedoeling dat in de OrderCollection de dropoffpoint als array terugkomt, dan zou ik de benaming van de method daar ook op toespitsen, nu was het in eerste instantie verwarrend.